mindmeld/ser.py

```diff
@@ -134,16 +141,18 @@ def parse_numerics(sentence, dimensions=None, language='EN', locale='en_US',
         'latent': True,
     }
 
-    valid_locales = ["en_AU", "en_BE", "en_BZ", "en_CA", "en_CN", "en_GB", "en_HK", "en_IE",
-                     "en_IN", "en_JM", "en_MO", "en_NZ", "en_PH", "en_TT", "en_TW", "en_US",
-                     "en_ZA"]
+    # Set the default locale to be en_us if the locale is not passed in and the langauge code
+    # is english
+    if not locale and language == 'EN':
+        locale = 'en_US'
+
     if locale:
-        if locale in valid_locales:
+        if locale in MINDMELD_LOCALES:
             data['locale'] = locale
         else:
             logger.error(
                 'Invalid locale provided, it should be from this set: %s. Ignoring argument.',
-                valid_locales)
+                MINDMELD_LOCALES)
```

Comment:
Is duckling going to throw an error? If so maybe we can just surface that instead of having our own hard-coded list that could easily get out of date.

Reply:
Duckling does not throw any errors, I think it defaults to EN:

```
VIJAYRK-M-L1E5:assistant-nlp vijay$ curl -XPOST http://0.0.0.0:7151/parse --data 'lang=PK&locale=en_KH&text=set the time to 10am' | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   763    0   717  100    46   277k  18203 --:--:-- --:--:-- --:--:--  350k
[
  {
    "body": "to 10",
    "start": 13,
    "value": {
      "value": 10,
      "type": "value",
      "unit": "degree"
    },
    "end": 18,
    "dim": "temperature",
    "latent": false
  },
  {
    "body": "10",
    "start": 16,
    "value": {
      "value": 10,
      "type": "value",
      "unit": "degree"
    },
    "end": 18,
    "dim": "temperature",
    "latent": false
  },
  {
    "body": "10",
    "start": 16,
    "value": {
      "value": 10,
      "type": "value"
    },
    "end": 18,
    "dim": "number",
    "latent": false
  },
  {
    "body": "10am",
    "start": 16,
    "value": {
      "values": [
        {
          "value": "2019-09-04T10:00:00.000-07:00",
          "grain": "hour",
          "type": "value"
        },
        {
          "value": "2019-09-05T10:00:00.000-07:00",
          "grain": "hour",
          "type": "value"
        },
        {
          "value": "2019-09-06T10:00:00.000-07:00",
          "grain": "hour",
          "type": "value"
        }
      ],
      "value": "2019-09-04T10:00:00.000-07:00",
      "grain": "hour",
      "type": "value"
    },
    "end": 20,
    "dim": "time",
    "latent": false
  }
]


```